### ISSUE TITLE
scr_namingscreen Decompilation Error (New Decompiler)

### ISSUE THREAD

**Issue by theclashingfritz** [krzys-h/UndertaleModTool#235]
This probably exists in the old one as well but in the new decompiler there's a couple of break sections that are left without an proper break in scr_namingscreen for entering you name.

The code should decompile to 
`
if ((selected_col < 0) || ((selected_row < 0) || (string_length(charmap[selected_row, selected_col]) > 0))) {
   break
} else {
    continue
}
`
but instead decompiles to 
`
if ((selected_col < 0) || ((selected_row < 0) || (string_length(charmap[selected_row, selected_col]) > 0))) {
} else {
    continue
}
`

**Comment by Grossley** [comment]
This might be related to #224 . Good find. :)

**Comment by theclashingfritz** [comment]
They are likely linked. Though in which ways is not yet known.

**Comment by Grossley** [comment]
So as it turns out this actually causes the game to freeze upon selecting "Begin Game". This needs to be corrected in the future (and I assume errors of this type will be addressed later on when the decompiler rewrite is done).

**Comment by Frisktaker** [comment]
Yep, ran into this same issue on 0.6.10. Had to downgrade back to my previous version (0.5.10, oddly enough), else my game was forever bricked due to the exact scenario Grossley mentioned, cuz no matter how many times I manually corrected it, saving would make it be like "hold on, let me screw this up again for you".

**Comment by Jacky720** [comment]
That doesn't sound correct. The displayed code will screw up again if profile mode isn't on, yes, but if you make the correction then the compiler should result in it working in-game.
